src/vivarium_testing_utils/automated_validation/data_transformation/measures.py

```diff
@@ -144,48 +143,68 @@ class Incidence(RatioMeasure):
     """Computes Susceptible Population Incidence Rate."""
 
     def __init__(self, cause: str) -> None:
-        self.measure_key = f"cause.{cause}.incidence_rate"
+        self.measure_key = self.artifact_key = f"cause.{cause}.incidence_rate"
         self.numerator = TransitionCounts(cause, f"susceptible_to_{cause}", cause)
         self.denominator = StatePersonTime(cause, f"susceptible_to_{cause}")
 
+    @check_io(artifact_data=SingleNumericColumn, out=SingleNumericColumn)
```

Comment:
If you are able to move everything to RatioMeasure this would also be able to be moved there.